### ISSUE TITLE
 Freeze parameters

### ISSUE THREAD

**Issue by 7rwang** [graphdeco-inria/gaussian-splatting#1083]
Hi,

is there anybody know how to freeze some parameters of 3d Gaussian like position or rotation etc. 

Just setting `requires_grad` False seems like not enough, cuz the position still changes even I set false

**Comment by leblond14u** [comment]
Hi, 

The quickest solution is to set a close to zero learning rate for these parameters if you only want to rapidly test things.
Otherwise the cleanest method is to get those parameters out of the gaussian optimizable parameters in the gaussian model file. 

Best,
